corehq/apps/reports/views.py

```diff
@@ -1856,6 +1856,7 @@ class TableauServerView(BaseProjectReportSectionView):
     page_title = gettext_lazy('Tableau Server Config')
     template_name = 'hqwebapp/crispy/single_crispy_form.html'
 
+    @cls_require_superuser_or_contractor
```

Comment:
You should also update tabclasses.py so people who don't have access don't see the link in the menu.